ReactiveCocoa/Swift/Atomic.swift

```diff
@@ -18,35 +18,40 @@ public final class Atomic<Value> {
 		get {
 			return withValue { $0 }
 		}
-	
 		set(newValue) {
 			modify { _ in newValue }
 		}
 	}
 	
 	/// Initializes the variable with the given initial value.
+	/// - parameter value: Value to be saved by the initialized atomical storage.
 	public init(_ value: Value) {
 		_value = value
 	}
 	
+	/// Try to held the lock of this atomic variable. If the lock is already held the
+	/// thread will wait in a loop ("spin") while repeatedly checking if the lock is
+	/// available.
 	private func lock() {
 		OSSpinLockLock(&spinLock)
 	}
 	
+	/// The running thread will realase the hold on this atomic variable lock.
 	private func unlock() {
 		OSSpinLockUnlock(&spinLock)
 	}
 	
-	/// Atomically replaces the contents of the variable.
-	///
-	/// Returns the old value.
+	/// Atomically replaces the contents of the stored variable.
+	/// - parameter newValue: Value to be stored.
+	/// - returns: The value previously stored.
```

Comment:
Like above, this (1) doesn't match the RAC style and (2) doesn't really enhance the documentation.